### DISCUSSION TITLE
Domain Verification Issue - Incorrect Subdomain Recognition

### DISCUSSION THREAD

**Discussion by gileadnetto** [vercel/community#3561] [Help]
### Summary

Dear Vercel Support,

I hope this message finds you well. I'm encountering an issue related to domain verification on the Vercel platform. I have successfully configured the TXT record for domain verification on the subdomain "_vercel.ime.usp.br" as instructed. The DNS propagation has also taken place, and the TXT record is fully propagated.

However, it seems that the Vercel platform is not recognizing the domain verification in the correct subdomain. Instead, it appears to be looking at the wrong subdomain, "_vercel.usp.br," which is causing a verification failure.

I'm reaching out to seek assistance in resolving this matter. I've followed the provided instructions carefully, and I'm confident that the configuration on my end is correct. It's crucial for me to get this domain properly verified and linked to my project on Vercel.

Could you please investigate this issue and guide me on the correct steps to ensure that Vercel recognizes the domain verification in the "_vercel.**ime**.usp.br" subdomain? If there are any additional actions I need to take or if there's a technical issue on the platform, your guidance will be highly appreciated.

Thank you for your attention and assistance in resolving this matter promptly.

https://mxtoolbox.com/SuperTool.aspx?action=txt%3A_vercel.ime.usp.br

![image](https://github.com/vercel/community/assets/23746381/2b4225bb-6603-4d5f-925c-d21f581d9b6b)

Best regards,

### Steps to Reproduce

![preview](https://github.com/vercel/community/assets/23746381/1755a118-6a07-4f9b-8120-148dbd2991d2)

prop... to _vercel.me.usp.br

![image2](https://github.com/vercel/community/assets/23746381/6f6a7276-3e92-4eb5-a8d0-231409fca1f0)

**Comment by amyegan** [comment]
Please add the record for `_vercel.usp.br` as instructed. This will verify domain ownership so that your specified subdomain can be assigned to the project.

**Reply by gileadnetto** [reply]
Dear Vercel Support,

Thank you for your prompt response. I appreciate your assistance. However, I'd like to clarify that due to the domain structure used by our organization, which is a federal university, the correct subdomain for domain verification is "_vercel.ime.usp.br" and not "_vercel.usp.br." Unfortunately, we do not have control over the "_vercel.usp.br" domain, as it is a distinct subdomain within the broader university domain.

The "_vercel.ime.usp.br" subdomain is under the control and administration of our university's informatics department (Instituto de Matemática e Estatística - IME) and is the appropriate place to configure domain-related settings for our projects.

I kindly request that the Vercel platform allow for domain verification on the "_vercel.ime.usp.br" subdomain to accommodate our organizational structure. This would greatly facilitate our ability to use the domain in our projects on Vercel.

If there's any additional information you need from us to facilitate this, please let us know. We greatly appreciate your understanding and support in this matter.

Thank you once again for your assistance.

Best regards,

Gilead Netto

**Reply by gileadnetto** [reply]
?
